src/components/view/QuickView.vue

```diff
@@ -0,0 +1,85 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <a-popover v-if="enabled && actionsExist" triggers="hover" placement="right">
+    <template slot="content">
+      <action-button
+        :size="size"
+        :actions="actions"
+        :dataView="true"
+        :resource="resource"
+        @exec-action="execAction" />
+    </template>
+    <a-icon type="right-circle" style="margin-left: 5px" />
```

Comment:
Could we use `<a-icon type="more" />` and  use float:right to keep it at right most side?

Reply:
Using the more icon doesn't look as good, it shows up very faint
Keeping it float right means that the user has to go to the rightmost end of the column to get the hover and back to the record name to get into the detail view. This way the icon is right after the name